simple/msg_sockets.c

```diff
@@ -306,6 +306,7 @@ static int setup_handle(void)
 	struct addrinfo *ai, aihints;
 	const char *bound_addr_str;
 	int ret;
+	char* saved_src_addr = 0;
```

Comment:
Please set pointers to NULL instead of 0, and if this initialization isn't needed (I don't think it is) just remove it.